language/stdlib/modules/LibraAccount.move

```diff
@@ -887,8 +887,8 @@ module LibraAccount {
         auth_key_prefix: vector<u8>,
         human_name: vector<u8>,
     ) {
-        assert(Roles::has_libra_root_role(creator_account), ENOT_LIBRA_ROOT);
```

Comment:
This is checked in `Roles::new_validator_role`?

Reply:
yep! Good catch. This is checked in roles (removed to be more in-line with the way checks are performed in the other `create_account` functions). I'll add a comment.